src/db/db.js

```diff
@@ -0,0 +1,10 @@
+require('dotenv').config();
+const { drizzle } = require('drizzle-orm/postgres-js');
```

Comment:
Now that ES6 modules are natively supported by NodeJS latest version, we can avoid tech debts: https://nodejs.org/api/packages.html#type

Reply:
this is done